qemu/slirp/slirp.c

```diff
@@ -951,7 +951,7 @@ static int if_encap6(Slirp *slirp, struct mbuf *ifm, struct ethhdr *eh,
  */
 int if_encap(Slirp *slirp, struct mbuf *ifm)
 {
-    uint8_t buf[1600];
+    uint8_t buf[65521 + 100]; // max MTU + 100
```

Comment:
MEMO: it looks like we can remove extra copy to this buf if we change the proto type of `slirp_output()`